code/style.css

```diff
@@ -1,4 +1,437 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+*{
+  box-sizing: border-box;
 }
+
+figure {
+  margin: 0;
+}
+
+.pagewraper{
+  max-width: 1300px;
+  margin-left: auto;
+  margin-right: auto;
+  width:100%;
+}
+
+.header {
+  background-color:white;
+  border-bottom: 2px solid grey;
+}
+
+.header__title{
+  font-size: 60px;
+  font-weight: bolder;
+  color: black;
+  margin: 0;
+  padding: 10px;
+  font-family:'Cormorant Garamond', serif;
+  text-decoration: none;
+  white-space:nowrap;
+  flex:1;
+}
+
+@media screen and (max-width: 900px){
```

Comment:
To structure the CSS styling a bit try to keep the media queries at the bottom of the file, makes it easier to keep the styling for each device separate and clean!